src/ddmd/dsymbolsem.d

```diff
@@ -3048,6 +3048,12 @@ extern(C++) final class DsymbolSemanticVisitor : Visitor
                 }
             }
         }
+        else if (pd.ident == Id.crt_constructor || pd.ident == Id.crt_destructor)
+        {
+            if (pd.args && pd.args.dim != 0)
+                pd.error("too many arguments");
```

Comment:
`too many` is misleading. Should be `arguments are not allowed for crt_constructor and crt_destructor functions`.

Reply:
These functions should also be forced to use C linkage.